MVP/defaults-build.yaml

```diff
@@ -12,18 +12,20 @@
             script-only-if-succeeded: False
             script-only-if-failed: True
             builders:
-              - shell: |
-                  #!/bin/bash
-                  echo "Jenkins job failed or canceled - canceling any submitted Beaker jobs"
-                  bkr job-cancel ${{BKR_JOBID}}
+                - shell: |
+                    echo "Jenkins job failed or canceled - canceling any submitted Beaker jobs"
+                    bkr job-cancel ${{BKR_JOBID}}
         - archive:
             artifacts: '*.txt, *.xml, *.properties'
             allow-empty: 'true'
         - junit:
             results: 'J:*.xml'
         - email-ext:
             recipients: ${{OWNERSHIP,var="JOB_COOWNERS_EMAILS"}}
-            subject: $DEFAULT_SUBJECT [${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}]
+            subject: $DEFAULT_SUBJECT [${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}] (${{TEST_COUNTS,var="fail"}}/${{TEST_COUNTS}})
+            body: |
+                $DEFAULT_CONTENT
+                Beaker job at https://beaker.engineering.redhat.com/matrix/?job_ids=${{ENV,var="BKR_JOBID_NUM"}}
```

Comment:
Disclosing internal URLs is discouraged by the Infosec policy (although this one is disclosed already elsewhere, and now even here :) ).

Consider referring to e.g. Jenkins-wide configuration environmental variable.